dpop-authz.md

```diff
@@ -0,0 +1,15 @@
+![ecosystem diagram](https://raw.githubusercontent.com/solid/app-authorization-panel/master/images/webid-dpop.png)
+
+## Verifying Application Identity
+
+* Authorization Servers MUST verify app identity (WebID), and include it in JWT issued as *access token*. *TODO: define mechanism - `redirect_uri`, public key for secure clients*
```

Comment:
I think the current Justin Richer proposal doesn't constrain it to WebIds.

Reply:
Ok, I'll remove mention of WebID. This point just assigns responsibility of verifying client identity to the Authorization Server and all the Resource Servers can just rely on information in the token.